### PR TITLE
Dev/xiowei/perf maintenance

### PR DESCRIPTION
**Pull request by vanbasten2323**
Added PerformMaintenance API. The RDFE API is called PerformMaintenance while the PS/CLI commands are called InitiateMaintenance. Also added MaintenanceStatus field in the GetDeployment response.

### CODE REVIEW

**Comment by huangpf** [comment]
@vanbasten2323 

The file shall be added to:
```
lib/services/computeManagement/lib/computeManagementClient.js
```

But currently it's:
```
lib/services/computeManagement/computeManagement/lib/computeManagementClient.js
```

**Comment by vanbasten2323** [comment]
This file is auto-generated, right?

**Comment by huangpf** [comment]
Yes, but you put it into a wrong folder... It can be easily spotted that there are 2 ```computeManagement```'s in the path...

**Comment by huangpf** [comment]
LGTM.

**Comment by huangpf** [comment]
@balajikris @amarzavery Could you please help review? Thanks.

File: lib/services/computeManagement/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "Lee, Hyonho <Hyonho.Lee@microsoft.com>",
     "Desai, Priyank <priydes@microsoft.com>"
   ],
-  "version": "0.17.1",
+  "version": "0.18.1",
```
Comment (huangpf) [inline lib/services/computeManagement/package.json]: 0.18.0

**Comment by vanbasten2323** [comment]
All fixed: put the computeManagementClient.js file to the correct location; Also changed the version in package.json from 0.18.1 to 0.18.0.

**Pull request merged**